e2e/kube/examples_test.go

```diff
@@ -170,33 +170,33 @@ var _ = Describe("Examples", func() {
 					By("Creating bosh deployment")
 					kubectlHelper := testing.NewKubectl()
 					err := testing.Create(namespace, yamlFilePath)
-					Expect(err).ToNot(HaveOccurred())
+					Expect(err).ToNot(HaveOccurred(), "error creating instance")
 
 					By("Checking for pods")
 					err = kubectlHelper.Wait(namespace, "ready", "pod/nats-deployment-nats-v1-0", kubectlHelper.PollTimeout)
-					Expect(err).ToNot(HaveOccurred())
+					Expect(err).ToNot(HaveOccurred(), "error waiting for pod/nats-deployment-nats-v1-0")
 
 					err = kubectlHelper.Wait(namespace, "ready", "pod/nats-deployment-nats-v1-1", kubectlHelper.PollTimeout)
-					Expect(err).ToNot(HaveOccurred())
+					Expect(err).ToNot(HaveOccurred(), "error waiting for pod/nats-deployment-nats-v1-1")
 
 					err = testing.RestartOperator(namespace)
-					Expect(err).ToNot(HaveOccurred())
+					Expect(err).ToNot(HaveOccurred(), "error restarting cf-operator")
 
 					By("Checking for pods not created")
 					err = kubectlHelper.Wait(namespace, "ready", "pod/nats-deployment-nats-v2-0", 10*time.Second)
 					Expect(err).To(HaveOccurred(), "error unexpected version of instance group is created")
 
 					By("Checking for secrets not created")
 					exist, err := kubectlHelper.SecretExists(namespace, "nats-deployment.bpm.nats-v2")
-					Expect(err).To(HaveOccurred())
+					Expect(err).ToNot(HaveOccurred(), "error getting secret/nats-deployment.bpm.nats-v2")
```

Comment:
Why was the expectation negated here and in the cases below?

Reply:
This case is that we don't want new versions to be created when the operator restart. I changed `kubectlHelper.SecretExists` that doesn't return an error when object not found because I want to use it in the WaitForSecret func that didn't break poll when object not found.

Reply:
Got it.